website/src/app/documentation/change-log/change-log.component.html

```diff
@@ -0,0 +1,129 @@
+<section class="dox-content">
```

Comment:
we moved the changelog to github to the releases page some time ago. pls use that place over there for this documentation (see https://github.com/porscheinformatik/clarity-addons/releases)